packages/workspace/src/utils/lint.ts

```diff
@@ -263,6 +269,12 @@ const globalESLint = JSON.stringify({
             ],
           },
         ],
+        '@nrwl/nx/no-barrel-files-import': [
+          'error',
+          {
+            disableImportFromParentAndCurrentRoot: false,
+          },
+        ],
```

Comment:
This part will add this rule for new projects, right?

Reply:
From what I understand - yes.